pkg/apis/configs/v1alpha1/configmappropagation_defaults.go

```diff
@@ -0,0 +1,31 @@
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	"context"
+)
+
+func (cmp *ConfigMapPropagation) SetDefaults(ctx context.Context) {
+	// If we haven't configured the original namespace,
+	// then set the default original namespace to the knative-eventing.
+	// TODO if this API starts to become commonly adopted by others,
+	//  it will stop make sense defaulting to knative-eventing.
+	if cmp != nil && cmp.Spec.OriginalNamespace == "" {
+		cmp.Spec.OriginalNamespace = "knative-eventing"
```

Comment:
I know it was already said, but Serving will also need this controller. Do we really want to set that default (Eventing)?

Reply:
Removed